src/AutoUI/Actions/Create.tsx

```diff
@@ -55,42 +57,46 @@ export const Create = <T extends AutoUIBaseResource<T>>({
 		disabledReasonsByAction[action.title];
 
 	return (
-		<Box>
-			<Button
-				data-action={`create-${model.resource}`}
-				onClick={() =>
-					onActionTriggered({
-						action,
-						schema: autoUIJsonSchemaPick(
-							model.schema,
-							model.permissions.create,
-						),
-					})
-				}
-				icon={<FontAwesomeIcon icon={faMagic} />}
-				tooltip={
-					typeof disabledReason === 'string' ? disabledReason : undefined
-				}
-				disabled={!!disabledReason}
-				primary
+		<Box display="flex">
+			<Tooltip
+				title={typeof disabledReason === 'string' ? disabledReason : undefined}
 			>
-				<ActionContent<T>
-					getDisabledReason={action.isDisabled}
-					affectedEntries={undefined}
-					checkedState={undefined}
-					onDisabledReady={(result) => {
-						setDisabledReasonsByAction((disabledReasonsState) => ({
-							...disabledReasonsState,
-							[action.title]: result,
-						}));
-					}}
+				<Button
+					data-action={`create-${model.resource}`}
+					variant="contained"
+					onClick={() =>
+						onActionTriggered({
+							action,
+							schema: autoUIJsonSchemaPick(
+								model.schema,
+								model.permissions.create,
+							),
+						})
+					}
+					startIcon={<AutoFixNormal />}
+					disabled={!!disabledReason}
 				>
-					<Flex justifyContent="space-between">
-						{action.title}
-						<Spinner ml={2} show={disabledReason === LOADING_DISABLED_REASON} />
-					</Flex>
-				</ActionContent>
-			</Button>
+					<ActionContent<T>
+						getDisabledReason={action.isDisabled}
```

Comment:
I know we had this already but what is it for if we already disable the button and show a tooltip when the action is disabled? 😕 If they're different, couldn't we put them together?

Reply:
Just looked at the component locally, now I'm wondering if we even need `ActionContent` here 🤔

Reply:
Not sure if I get the comment, ActionContent is a component that has an async function and it is needed to respect a specific structure, I think that the explanation in the comment should be enough https://github.com/balena-io-modules/autoui/blob/22ba138681821f623b1e69d00a0a60dff9cb1bfa/src/AutoUI/Actions/ActionContent.tsx#L14